components/header.js

```diff
@@ -1,25 +1,26 @@
-import React, { Component } from "react";
-import ReactDOM from "react-dom";
+import React, {Component} from "react";
 
-import Search from "./search"
+import Search from "./search";
+import AddNewBoard from "../containers/add_board_container";
 
-export class Header extends Component {
-    constructor(props) {
-        super(props);
+export default class Header extends Component {
+  constructor(props) {
```

Comment:
there's no need for this constructor and props since you are not using them. 
Add a linter please.